django/pyrog/api/serializers.py

```diff
@@ -4,12 +4,16 @@
 
 
 class SourceSerializer(serializers.ModelSerializer):
+    resources = serializers.PrimaryKeyRelatedField(many=True, read_only=True)
```

Comment:
:(

Reply:
can we use
```
class Meta:
    fields = [..., "resources"]
```
instead ? this way we don't need to specify a serializer